osu.Game/Screens/Play/Player.cs

```diff
@@ -69,9 +72,12 @@ public class Player : OsuScreen
         private bool loadedSuccessfully => RulesetContainer?.Objects.Any() == true;
 
         [BackgroundDependencyLoader(permitNulls: true)]
-        private void load(AudioManager audio, OsuConfigManager config, OsuGame osu)
+        private void load(AudioManager audio, OsuConfigManager config, OsuGame osu, APIAccess api)
         {
+            this.api = api;

```

Comment:
Since permitNulls is true, we can still reach the previous scenario here, but we'll only be able to notice it upon onCompletion. Can we throw an exception here if this is the case?